environment.yml

```diff
@@ -30,6 +30,7 @@ dependencies:
   - pandas=1.5.1
   - pip=22.3
   - pygrib=2.1.4
+  - gcsfs=2022.11.0
```

Comment:
Is this supposed to match the setup.py? In other words, do you need xarray-beam here?

Reply:
I didn't know xarray-beam was on `conda`! Thanks.